code/websitesrc/cybertoolwebsite/amplify/backend/types/amplify-dependent-resources-ref.d.ts

```diff
@@ -1,4 +1,5 @@
 export type AmplifyDependentResourcesAttributes = {
+<<<<<<< Updated upstream
```

Comment:
Looks like committed merge conflicts. Guessing this is not what you want?

Reply:
Anything in the amplify/* director is subject to change and expected to conflict. Everytime you have to run the website locally anyway, you have to run the appropriate amplify pull command given by the AWS website. This command can and will overwrite/create files. If anything we can put the entire amplify directory in gitignore.